src/ActiveLogin.Authentication.BankId.AspNetCore/Models/BankIdLoginOptions.cs

```diff
@@ -5,13 +5,20 @@ namespace ActiveLogin.Authentication.BankId.AspNetCore.Models
 {
     public class BankIdLoginOptions
     {
-        public BankIdLoginOptions(List<string> certificatePolicies, SwedishPersonalIdentityNumber personalIdentityNumber, bool allowChangingPersonalIdentityNumber, bool autoLaunch, bool allowBiometric)
+        public BankIdLoginOptions(
+            List<string> certificatePolicies,
+            SwedishPersonalIdentityNumber personalIdentityNumber,
+            bool allowChangingPersonalIdentityNumber,
+            bool autoLaunch,
+            bool allowBiometric,
+            bool idUseQrCode)
```

Comment:
I think "Bank" fell of here.

Reply:
It did. Looking at it with fresh eyes I think we remove the `id` part as well. The context `BankId` is given by the class.